
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("DATA/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'DATA/00006695_aaaaaadg_s000_0hne_0000_a003_lvl003_t000_RGBAVALUES.csv'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("EVAL_DATA2/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'EVAL_DATA2/00006695_aaaaaadg_s000_0hne_0000_a003_lvl003_t000_RGBAVALUES.csv'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("TRAIN_DATA2/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'EVAL_DATA2/00006695_aaaaaadg_s000_0hne_0000_a003_lvl003_t000_RGBAVALUES.csv'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 14, in main
    svs_list = read_file_lists(svs_list_file)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 9, in read_file_lists
    df = pandas.read_csv(file_name)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1024, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 618, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1618, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1878, in _make_engine
    self.handles = get_handle(
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/data/isip/data/tuh_dpath_breast/deidentified/v3.0.0/svs/eval/tuh_im_train.list'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 14, in main
    svs_list = read_file_lists(svs_list_file)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 9, in read_file_lists
    df = pandas.read_csv(file_name)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1024, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 618, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1618, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1878, in _make_engine
    self.handles = get_handle(
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/data/isip/data/tuh_dpath_breast/deidentified/v3.0.0/svs/eval/tuh_im_train.list'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 14, in main
    svs_list = read_file_lists(svs_list_file)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 9, in read_file_lists
    df = pandas.read_csv(file_name)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1024, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 618, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1618, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1878, in _make_engine
    self.handles = get_handle(
  File "/data/isip/exp/tuh_dpath/exp_0280/venv/lib/python3.9/site-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '/data/isip/data/tuh_dpath_breast/deidentified/v3.0.0/svs/eval/tuh_im_train.list'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("TRAIN_DATA2/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'TRAIN_DATA2/00477780_aaaaaagg_s000_0hne_0000_a005_lvl001_t000_RGBAVALUES.csv'
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("TRAIN_DATA2/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'EVAL_DATA2/00006695_aaaaaadg_s000_0hne_0000_a003_lvl003_t000_RGBAVALUES.csv'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
Traceback (most recent call last):
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 21, in <module>
    main()
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py", line 17, in main
    nedc_regionid.classify_frames(svs,csv,window_size,frame_size)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_regionid.py", line 88, in classify_frames
    nedc_fileio.RGBA_to_dct(window_list,imagefile)
  File "/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/../lib/nedc_fileio.py", line 274, in RGBA_to_dct
    file = open("TRAIN_DATA2/"+ifile+"_RGBAVALUES.csv",'w')
FileNotFoundError: [Errno 2] No such file or directory: 'TRAIN_DATA2/00477780_aaaaaagg_s000_0hne_0000_a005_lvl001_t000_RGBAVALUES.csv'
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
/home/tuo54571/Machine-Learning-Applications-In-Digital-Pathology/nedc_mladp/src/gen_feats.py:4: DeprecationWarning: 
Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
(to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
but was not found to be installed on your system.
If this would cause problems for you,
please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
        
  import pandas
